src/component/mxgraph/shape/activity-shapes.ts

```diff
@@ -75,6 +77,23 @@ export abstract class BaseActivityShape extends mxRectangleShape {
       });
     }
   }
+
+  private getIconOriginForMarkerIcon(allMarkers: string[], markerOrder: number): (canvas: BpmnCanvas) => void {
```

Comment:
:question: why not passing only the markers length as the marker names are not used here

Reply:
👍